mlir/test/Target/SPIRV/function-decorations.mlir

```diff
@@ -17,3 +17,37 @@ spirv.module Logical GLSL450 requires #spirv.vce<v1.0, [Shader, Linkage], []> {
     }
     spirv.func @inside.func() -> () "Pure" attributes {} {spirv.Return}
 }
+
+// -----
```

Comment:
Are the `// check` lines below even executed when not in upper case?

Reply:
Sorry..these were in lower case by mistake.